lib/card/content.rb

```diff
@@ -26,7 +26,7 @@ def card
     end
     
     def chunk_list
-      @opts[:chunk_list] || card.chunk_list
+      @opts[:chunk_list] || @format.chunk_list
```

Comment:
that's a notable change. I moved chunk_list from the Card class to the Format class.